recipe/conda_build_config.yaml

```diff
@@ -159,6 +173,24 @@ docker_image:                                   # [os.environ.get("BUILD_PLATFOR
   - quay.io/condaforge/linux-anvil-cuda:11.8              # [ppc64le and os.environ.get("CF_CUDA_ENABLED", "False") == "True" and os.environ.get("BUILD_PLATFORM") == "linux-64"]
   - quay.io/condaforge/linux-anvil-cuda:11.8              # [aarch64 and os.environ.get("CF_CUDA_ENABLED", "False") == "True" and os.environ.get("BUILD_PLATFORM") == "linux-64"]
 
+  # CUDA 12.0
+  - quay.io/condaforge/linux-anvil-cos7-x86_64     # [linux64 and os.environ.get("BUILD_PLATFORM") == "linux-64" and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  # case: native compilation (build == target)
+  - quay.io/condaforge/linux-anvil-ppc64le         # [ppc64le and os.environ.get("BUILD_PLATFORM") == "linux-ppc64le"]
+  - quay.io/condaforge/linux-anvil-aarch64         # [aarch64 and os.environ.get("BUILD_PLATFORM") == "linux-aarch64"]
+  # case: cross-compilation (build != target)
+  - quay.io/condaforge/linux-anvil-cos7-x86_64     # [ppc64le and os.environ.get("BUILD_PLATFORM") == "linux-64"]
+  - quay.io/condaforge/linux-anvil-cos7-x86_64     # [aarch64 and os.environ.get("BUILD_PLATFORM") == "linux-64"]
```

Comment:
I'm also pretty sure that the selectors here need `os.environ.get("CF_CUDA_ENABLED", "False") == "True"` (which was active in the migrator through the selector on `docker_images`).